documentation/modules/creating-vddk-image.adoc

```diff
@@ -35,11 +35,6 @@ $ mkdir /tmp/<dir_name> && cd /tmp/<dir_name>
 
 . In a browser, navigate to the link:https://developer.vmware.com/web/sdk/8.0/vddk[VMware VDDK version 8 download page].
 . Select version 8.0.1 and click *Download*.
-+
-[NOTE]
-====
-In order to migrate to {virt} 4.12, download VDDK version 7.0.3.2 from the link:https://developer.vmware.com/web/sdk/7.0/vddk[VMware VDDK version 7 download page].
-====
```

Comment:
as I commented on slack, we better keep this note as long as 4.12 is supported - how about adding a comment to remove it once 4.12 becomes unsupported instead?

Reply:
Done.